### ISSUE TITLE
Viewing README crashes app

### ISSUE THREAD

**Issue by doublah** [slapperwan/gh4a#846]
Crashes upon clicking README at https://github.com/tmm2018/TMM_QML , supposedly due to markdown.

Using latest app version on F-Droid.

**Comment by maniac103** [comment]
Clicking readme as in opening it from the repository overview or as in opening it from the file list?

**Comment by maniac103** [comment]
Hmm, works fine for me using a build off current master branch. I'll double check with 4.3.1; we probably fixed something I'm not aware of currently.

**Comment by goldfndr** [comment]
Yes, I can reproduce, error in utils/HttpImageGetter. loadImageForUrl in 4.3.1.
